components/datadog/fakeintake/component.go

```diff
@@ -9,7 +9,8 @@ import (
 type FakeintakeOutput struct { // nolint:revive, We want to keep the name as <Component>Output
 	components.JSONImporter
 
-	URL string `json:"url"`
+	ClientURL string `json:"clientURL"`
+	URL       string `json:"url"`
```

Comment:
Ideally when the URL used internally by Pulumi and the on used by the fakeintake client in the test is different we'd like to export only the URL used by the fakeintake client.
In this case we'd like Pulumi to use `fakeintake.default.svc.cluster.local` but we'd like `localhost:30080` to be exported as this the URL that must be used when initializing the fakeintake client.

Open to any suggestions

Reply:
Interesting, what is the URL you use `fakeintake.default.svc.cluster.local` to configure the agent and `ClientURL` for the fakeintake client? Let's discuss about this

Reply:
I will check if I can have something working using `<host_ip>:<node_port>` so I would not need to have 2 different URL.
Otherwise I will go with the `InternalURL`
Thanks!

Reply:
Updated the code to use the host ip to communicate with the fakeintake. It should ease things if we want to have other local providers. And we no longer need two different URL